src/styles/blocks/main.scss

```
@@ -0,0 +1,16 @@
+@import "./utils/_variables";
+
+.main {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding-top: 50px;
+  padding-bottom: 70px;
+
+  &__card {
+    max-width: $card-count *
```

Better use simply width, because according to the task we can't use max-width.
```suggestion
  &__catalog {
    width: $card-count *
```

---

Зробила